### PR TITLE
Enhancement: [dev-8903] dropdown styling

### PR DESCRIPTION
**Pull request by joshlacey**
## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

### CODE REVIEW

File: packages/core/styles/filters.scss [hunk 1]
```diff
@@ -1,11 +1,4 @@
 .filters-section {
-  &__wrapper {
```
Comment (adamdoe) [inline packages/core/styles/filters.scss:2]: Isn't this needed when titles are above filters? I'm not seeing any space above filters now.

File: packages/core/components/MultiSelect/multiselect.styles.css [hunk 2]
```diff
@@ -1,5 +1,7 @@
 .cove-multiselect {
+  --select-height: calc(1.5em + 0.75rem + 2px);
```
Comment (adamdoe) [inline packages/core/components/MultiSelect/multiselect.styles.css:2]: Can you add a comment for why this calculation is being used?

File: packages/core/components/MultiSelect/MultiSelect.tsx [hunk 3]
```diff
@@ -67,77 +77,86 @@ const MultiSelect: React.FC<MultiSelectProps> = ({ section = null, subsection =
...
+          <div
+            id={multiID}
+            onClick={() => {
+              if (!selectedItems.length) {
+                setExpanded(true)
+              }
             }}
+            className='selected'
           >
-            <Icon display={expanded ? 'caretDown' : 'caretUp'} style={{ cursor: 'pointer' }} />
```
Comment (adamdoe) [inline packages/core/components/MultiSelect/MultiSelect.tsx:107]: I'm assuming they don't want the caretUp here anymore for ux reasons.

File: packages/core/components/ui/Icon.tsx [hunk 4]
```diff
@@ -88,15 +88,24 @@ type IconProps = {
   /* Returns icon data as plain svg */
   base?: boolean
   /* Sets alt text for the icon */
-  alt: string
+  alt?: string
```
Comment (adamdoe) [inline packages/core/components/ui/Icon.tsx:91]: We might want to keep this requirement for 508.

File: packages/core/components/MultiSelect/MultiSelect.tsx (hunk 3 above)
Comment (joshlacey) [reply packages/core/components/MultiSelect/MultiSelect.tsx:107]: yes that was as per the prototype.

File: packages/core/components/ui/Icon.tsx (hunk 4 above)
Comment (joshlacey) [reply packages/core/components/ui/Icon.tsx:91]: I think having a different description of a checkmark throughout the app might be worse than defaulting to checkmark. I could create a new lookup with default names for each of the icons? so it doesn't end up as 'caretUp'. 

`const defaultNameLookup = { caretUp: 'caret up'}`

`<IconObj title={ alt || defaultNameLookup[display] } />`

if you think this is necessary. I just kept seeing a lot of places where the alt wasn't given and we had typescript errors.

File: packages/core/styles/filters.scss (hunk 1 above)
Comment (joshlacey) [reply packages/core/styles/filters.scss:2]: Can you screenshot what you're seeing? In Pages/Dashboard/Regressions API Filters Map it looks like there's sufficient space?

**Review by adamdoe — APPROVED** [review]

**Pull request merged**